openstudiocore/src/shared_gui_components/MeasureManager.cpp

```diff
@@ -199,7 +201,8 @@ std::vector<ruleset::OSArgument> MeasureManager::getArguments(analysisdriver::Si
     if (appmodel)
     {
       model = appmodel;
-      idf = boost::none;

```

Comment:
These lines of code are in the context of MeasureManager::getArguments which is called by MeasureManager::updateMeasure(s).  The update is kicked off when the scripts tab is shown in OpenStudio App.  https://github.com/NREL/OpenStudio/blob/1348-IDFMeasureWorkspace/openstudiocore/src/openstudio_lib/ScriptsTabView.cpp#L92